components/operators/keycloak/instance/overlays/ocp-oidc-client/patch-realm.yaml

```diff
@@ -0,0 +1,9 @@
+- op: replace
+  path: /spec/realm/clients/3/redirectUris
+  value: 
+    - 'http://localhost:8080/*'
+    # - 'https://keycloak-composer-ai-keycloak.apps-crc.testing/*'  
```

Comment:
Should we be using this comment value by default? And then commenting out localhost so it works when deployed to openshift?

Reply:
This redirect url here would be the location of the redirect for the app trying to login.  It probably wouldn't hurt to change it to something like quarkus-llm-router-composer-ai-keycloak.apps-crc.testing for better clairity.